libc/watcom/asm/segments.asm

```diff
@@ -107,57 +58,4 @@ _End_YI label byte
         public  "C",_End_YI
 YIE     ends
 
-_BSS    segment word public 'BSS'
-        ;extrn   _edata                  : byte  ; end of DATA (start of BSS)
-        ;extrn   _end                    : byte  ; end of BSS (start of STACK)
-_BSS    ends
-
-;STACK_SIZE      equ     1000h
-;STACK   segment para stack 'STACK'
-        ;db      (STACK_SIZE) dup(?)
-;STACK   ends
-
-        assume  nothing
-        public  _cstart_
-
-        assume  cs:_TEXT
-
-_cstart_ proc near
-        dw      __begtext               ; make sure dead code elimination
-                                        ; doesn't kill BEGTEXT segment
-_cstart_ endp
-
-if 0
-        INIT_VAL        equ 0101h
-        NUM_VAL         equ 16
-NullAssign      db      '*** NULL assignment detected',0
-;       don't touch AL in __exit, it has the return code
-__exit  proc
-        public  "C",__exit
-        push    ax                      ; save return code on stack
-        mov     dx,DGROUP
-        mov     ds,dx
-        cld                             ; check lower region for altered values
-        lea     di,__nullarea           ; set es:di for scan
-        mov     es,dx
-        mov     cx,NUM_VAL
-        mov     ax,INIT_VAL
-        repe    scasw
-        je      L7
-;
-; low memory has been altered
-;
-        pop     bx                      ; restore return code
-        mov     ax,offset NullAssign    ; point to msg
-        mov     dx,cs                   ; . . .
-
-; input: ( char __far *msg, int rc ) always in registers
-;       DX:AX - far pointer to message to print
-;       BX    - exit code
-
-__exit  endp
-endif
```

Comment:
I left all this code in to remind me of how OWC handles checking whether a NULL pointer has been referenced. I would also like this left in as this is under development but not finished yet.

Reply:
Rejected. Please leave `if 0` code in as I plan on using it later on.